src/components/tutorial-meta/components/badges/index.tsx

```diff
@@ -31,7 +31,7 @@ export function Badges({ options }: BadgesProps): React.ReactElement {
 
   return (
     <ul className={s.list}>
-      <Badge type="readTime" />
+      <p className={s.readTime}>{getReadableTime(readTime)}</p>
       {isBeta ? <Badge className={s.beta} type="isBeta" /> : null}
```

Comment:
There's still opportunity to refactor to use the global `Badge` component here, but not addressing that in this PR - https://app.asana.com/0/1202001387788447/1202016460311398